js/src/sheet.js

```diff
@@ -112,8 +112,8 @@ var SheetModel = widgets.DOMWidgetModel.extend({
     },
     _cell_data_to_grid: function(cell, data) {
         var value = cell.get('value');
-        if(!value)
-            return
+        // if(!value)
+        //     return
```

Comment:
I think the check should have been if it's null (and maybe undefined), now indeed 0 is also not shown. It would be great if you can include a test for this, similar to:
https://github.com/QuantStack/ipysheet/blob/d2705f8480b6c033cbca2d5812ec550a983f19da/js/test/sheet.ts#L156
Let me know if you are familiar with this, or want to get a few pointers.

Reply:
In my demo notebook it worked (see notebooks folder), but you're right the more specific the better. So I checked value for null or undefined.  
I also added a corresponding test - pls check as it's the first time I write one ! - I know I should not be proud of it...